src/fe-common/core/fe-common-core.c

```diff
@@ -462,25 +463,59 @@ void fe_common_core_finish_init(void)
 gboolean strarray_find_dest(char **array, const TEXT_DEST_REC *dest)
 {
 	g_return_val_if_fail(array != NULL, FALSE);
-
-	if (strarray_find(array, "*") != -1)
+	const char *type = module_find_id_str("WINDOW ITEM TYPE", dest->window->active->type);
+
+	if ((strarray_find(array, "*") != -1) || // we ignore all targets
```

Comment:
Put the check for `*` in its own line please (and comments on the line above the one they refer to)